### PR TITLE
telco5g/cnf-tests: Set CNF_BRANCH var

### PR DESCRIPTION
**Pull request by sabinaaledort**
Doing this so we can checkout the right branch based on the tested version.

### CODE REVIEW

**Comment by sabinaaledort** [comment]
/hold
need to cherry-pick changes in cnf repo to release-4.11

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml [hunk 1]
```diff
@@ -619,6 +619,9 @@ tests:
   cluster: build05
   cron: 21 22 */1 * 1-5
   steps:
+    env:
+      E2E_TESTS_CONFIG: |
```
Comment (sshnaidm) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:623]: Why just not to use `CNF_BRANCH: release-4.11` ?

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml [hunk 2]
```diff
@@ -622,6 +622,9 @@ tests:
   cluster: build05
   cron: 21 3 */1 * 1-5
   steps:
+    env:
+      E2E_TESTS_CONFIG: |
```
Comment (sshnaidm) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:626]: same here

File: ci-operator/step-registry/telco5g/cnf-tests/telco5g-cnf-tests-commands.sh [hunk 3]
```diff
@@ -139,6 +139,17 @@ export CNF_BRANCH="${CNF_BRANCH:-master}"
 
 echo "************ telco5g cnf-tests commands ************"
 
+if [[ -n "${E2E_TESTS_CONFIG:-}" ]]; then
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/cnf-tests/telco5g-cnf-tests-commands.sh:142]: And then no need in this section

**Comment by sshnaidm** [comment]
See how it's done in examples: https://docs.ci.openshift.org/docs/architecture/step-registry/#parameters

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml (hunk 1 above)
Comment (sabinaaledort) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:623]: than you need to change the env config to CNF_BRANCH [here](https://github.com/openshift/release/pull/32086/files#diff-ffd8a5dab2ba9b8eb035e5a4dd1ed4db6f043c718d3a04cca08a458d10fca674R13) and i was thinking to make it more generic so we can add more env variables in the future.

Comment (sshnaidm) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:623]: ok, fair enough

**Pull request merged**